cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -84,13 +96,79 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
                             "$nodeBuildDir/network-parameters:/opt/corda/network-parameters",
                             "$nodeBuildDir/additional-node-infos:/opt/corda/additional-node-infos",
                             "$nodeBuildDir/drivers:/opt/corda/drivers"
-
                     ),
                     "ports" to listOf(it.rpcPort, it.config.getInt("sshd.port")),
-                    "image" to "corda/corda-zulu-${it.runtimeVersion().toLowerCase()}"
+                    "image" to (dockerImage ?: "corda/corda-zulu-${it.runtimeVersion().toLowerCase()}")
             )
-        }.toMap()
 
+            if (dockerConfig.isNotEmpty()) {
+
+                var dockerConfig = ConfigFactory.parseMap(dockerConfig)
```

Comment:
We never change the value of `dockerConfig` once it's been assigned, so
```kotlin
val dockerConfig = ConfigFactory.parseMap(dockerConfig)
```
is better.